src/applications/hca/components/FormPages/DependentInformation.jsx

```diff
@@ -1,29 +1,61 @@
 import React, { useEffect, useMemo, useState } from 'react';
 import PropTypes from 'prop-types';
-
 import { VaModal } from '@department-of-veterans-affairs/component-library/dist/react-bindings';
 import { focusElement } from 'platform/utilities/ui';
 import { getActivePages } from 'platform/forms-system/src/js/helpers';
 import FormNavButtons from 'platform/forms-system/src/js/components/FormNavButtons';
-
 import DependentListLoopForm from '../FormFields/DependentListLoopForm';
 import useAfterRenderEffect from '../../hooks/useAfterRenderEffect';
 import {
   getDataToSet,
   getSearchAction,
   getSearchIndex,
   getDefaultState,
-} from '../../utils/helpers/listloop-pattern';
+  replaceStrValues,
+  canHaveEducationExpenses,
+} from '../../utils/helpers';
 import {
   DEPENDENT_VIEW_FIELDS,
-  DEPENDENT_SUBPAGES,
   SESSION_ITEM_NAME,
   SHARED_PATHS,
+  LAST_YEAR,
 } from '../../utils/constants';
+import content from '../../locales/en/content.json';
 
 // declare shared data & route attrs from the form
 const { dependents: DEPENDENT_PATHS } = SHARED_PATHS;
 
+// declare subpage configs for dependent information page
```

Comment:
This needed moved out of  the `Constants` file, as it created a dependency cycle after changing the imports. This is only used in this file, so it makes sense for it to just live here.